spec/lib/player_spec.rb

```diff
@@ -0,0 +1,104 @@
+require './lib/scrabble/player'
+
+describe "Player class" do
```

Comment:
Generally, if your specfile is for unit testing a single class, you'd put the name of the class in the describe here, without quotes, i.e. `describe Scrabble::Player do`. This gives you access to helper methods in rspec such as `described_class`.